bag/db/script/bag-db.sql

```diff
@@ -306,7 +306,7 @@ CREATE TABLE gemeente_woonplaats (
   gid serial,
   begindatumtijdvakgeldigheid TIMESTAMP WITH TIME ZONE,
   einddatumtijdvakgeldigheid TIMESTAMP WITH TIME ZONE,
-  woonplaatscode numeric(4),
+  woonplaatscode varchar(16),
```

Comment:
Heel strict genomen is de woonplaatscode altijd 4 cijfers, als in [XSD definitie](https://github.com/nlextract/NLExtract/blob/master/bag/doc/schema/imbag/imbag-types/v20090901/BagvsImbagOnderdelen-0.7.xsd#L151), dus varchar(4). Voorloopnullen voor woonplaatsen ben ik nooit tegengekomen, zie bijv http://www.metatopos.eu/plaatscodes.html maar er zijn meer bronnen. 
Wel voorloopnullen dus voor gemeenten, maar zag dat die met `LPAD` elders afgehandeld worden.